content/productivity/distribute-silver/plugin.js

```diff
@@ -137,15 +220,15 @@ function distributeSilver(fromId, maxDistributeEnergyPercent, minPLevel) {
     // Remember its a tuple of candidates and their distance
     const candidate = candidates_[i++][0];
 
-    // Rejected if has unconfirmed pending arrivals
+    // Rejected if has more than 5 unconfirmed pending arrivals
     const unconfirmed = df.getUnconfirmedMoves().filter(move => move.to === candidate.locationId)
-    if (unconfirmed.length !== 0) {
+    if (unconfirmed.length > 4) {
       continue;
     }
 
-    // Rejected if has pending arrivals
+    // Rejected if has more than 5 pending arrivals
     const arrivals = getArrivalsForPlanet(candidate.locationId);
-    if (arrivals.length !== 0) {
+    if (arrivals.length > 4) {
       continue;
     }
```

Comment:
these have to actually be combined into 1 call if you are doing it like this, otherwise you might end up with 10 attempts and the 5 pending will all revert.

Should just be able to combine add the lengths together and then do the >5

Reply:
Yeah just did that thanks. It took me a while to realize you could not send more than 5 tx to a single planet.